### PR TITLE
enhancement/s3JUnitTests

### LINKED ISSUE #39: Create Test for Test Architecture
Tests:
- [x] UserTest.java
- [x] LisitingTest.java
- [x] ItemDescriptionTest.java
- [x] ListItemAdapterTest.java

### PR DESCRIPTION
**Pull request by EvanMateo7**
Travis CI should already be running test commands ( i.e. :app:testDebugUnitTest ) for the test files.
Closes #39

### CODE REVIEW

**Comment by melodyvibe** [comment]
Closes issues #47

File: app/src/testDebug/java/com/projectfirebase/soen341/root/ItemDescriptionTest.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        assertEquals(itemTest.getDescription(), description);
+    }
+
+
+    @Test
+    public void setURL() throws Exception{
+        String url = "wabadabadingdong";
+        itemTest.setURL(url);
+        assertEquals(itemTest.getURL(), url);
+    }
```
Comment (kkhh001) [inline app/src/testDebug/java/com/projectfirebase/soen341/root/ItemDescriptionTest.java:111]: Clean code, easy to read 👍

**Review by kkhh001 — APPROVED** [review]
I'm not seeing anything to be enhanced, good job!

**Comment by kwreen** [comment]
Don't merge this pull request yet. My test for the ProfileFragment isn't exactly working, I did a tmp commit because my laptop died and I needed to keep working on it. Or you could just remove my changes and I'll work on a new branch.

**Comment by EvanMateo7** [comment]
@kwreen We'll add your test from another branch.

**Review by LHuss — APPROVED** [review]

**Pull request merged**